scalding-core/src/main/scala/com/twitter/scalding/serialization/RequiredBinaryComparatorsConfig.scala

```diff
@@ -2,6 +2,13 @@ package com.twitter.scalding.serialization
 
 import com.twitter.scalding.{ Config, Job }
 
+sealed trait RequireOrderedSerializationMode
+object RequireOrderedSerializationMode {
+  case object Fail extends RequireOrderedSerializationMode
+  case object Log extends RequireOrderedSerializationMode
+}
+
 trait RequiredBinaryComparatorsConfig extends Job {
-  override def config = super.config + (Config.ScaldingRequireOrderedSerialization -> "true")
+  def requireOrderedSerializationMode: RequireOrderedSerializationMode = RequireOrderedSerializationMode.Fail
+  override def config = super.config + (Config.ScaldingRequireOrderedSerialization -> requireOrderedSerializationMode.toString)
```

Comment:
What's the `.toString` supposed to output here? Aren't we matching on lowercase in `getRequireOrderedSerializationMode ` above?